addons/website/views/snippets/s_image_frame.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_frame" name="Image - Frame">
+    <section class="s_image_frame o_cc o_cc2 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item g-height-4 g-col-lg-5 col-lg-6 rounded-2 o_colored_level o_cc o_cc1" style="z-index: 1; grid-area: 7 / 7 / 11 / 12; --grid-item-padding-x: 36px; --grid-item-padding-y: 36px;">
+                    <h3>Product highlight</h3>
+                    <p class="lead">Choose a vibrant image and write an inspiring paragraph about it. It does not have to be long, but it should reinforce your image.</p>
+                </div>
+                <div class="o_grid_item o_grid_item_image d-none d-md-block g-height-12 g-col-lg-6 col-lg-6 rounded-2 overflow-hidden oe_img_bg o_colored_level o_cc o_cc1 " style="grid-area: 1 / 1 / 12 / 13; --grid-item-padding-x: 0px; --grid-item-padding-y: 0px;">
+                    <img src="/web/image/website.s_image_frame_default_image" class="img img-fluid mx-auto" alt=""/>
```

Comment:
To be tested but I'd switch the image to a background one, to fix the white padding in columns layout. 

Doing that will also allow you to add padding on both element, ensuring a better visual result in columns mode.

If the aspect-ratio of the image affects the layout, ignore this comment.

<img width="1134" alt="image" src="https://github.com/user-attachments/assets/2287af0e-faab-470e-87fb-4bfe66aa91f4">